config/crd/bases/multicluster.k8s.io_works.yaml

```diff
@@ -0,0 +1,186 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.5
+  creationTimestamp: null
+  name: works.multicluster.k8s.io
+spec:
+  group: multicluster.k8s.io
```

Comment:
realistically, we should start with x-k8s.io while we're working on this.  I don't think we're looking for api-review/approval this early.

Reply:
Agree.

Reply:
right, changed to multicluster.x-k8s.io